lfs/lfs.go

```diff
@@ -68,11 +68,12 @@ func ObjectExistsOfSize(oid string, size int64) bool {
 }
 
 func Environ() []string {
+	manifest := transfer.ConfigureManifest(transfer.NewManifest(), config.Config)
```

Comment:
I think this method should take at least a `Config` and maybe a `Manifest` as well, otherwise it could well be printing something entirely different to what is being used in practice.